home-manager/home-manager

```diff
@@ -82,18 +82,19 @@ function setFlakeAttribute() {
 
     if [[ -n "$FLAKE_ARG" ]]; then
         local flake="${FLAKE_ARG%#*}"
+        local name
         case $FLAKE_ARG in
             *#*)
-                local name="${FLAKE_ARG#*#}"
+                name="${FLAKE_ARG#*#}"
             ;;
             *)
-                local name="$USER@$(hostname)"
+                name="$USER@$(hostname)"
                 if [ "$(nix eval "$flake#homeConfigurations" --apply "x: x ? \"$name\"")" = "false" ]; then
                     name="$USER"
                 fi
             ;;
         esac
-        export FLAKE_CONFIG_URI="$flake#homeConfigurations.\"$name\""
```

Comment:
Except that this breaks the `$USER@HOSTNAME` I think

Reply:
The issue is this will break when used with a username containing dots